### PR TITLE
Swap 1039 proposal statuses

### PR DESCRIPTION
**Pull request by martin-trajanovski**

### CODE REVIEW

**Comment by jekabs-karklins** [comment]
@martin-trajanovski I believe the pipeline is failing because of known workflow issues. Here is a pullrequest for the fix. https://github.com/UserOfficeProject/user-office-frontend/pull/14

If your branch has the changes than there is a good chance that pipelines will pass.

**Review by peter-asztalos — APPROVED** [review]

**Review by jekabs-karklins — APPROVED** [review]
Looks good

File: src/components/settings/proposalStatus/CreateUpdateProposalStatus.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  close: (proposalStatusAdded: ProposalStatus | null) => void;
+  proposalStatus: ProposalStatus | null;
+};
+
+const CreateUpdateProposalStatus: React.FC<CreateUpdateProposalStatusProps> = ({
+  close,
+  proposalStatus,
+}) => {
+  const classes = useStyles();
+  const api = useDataApi();
```
Comment (jekabs-karklins) [inline src/components/settings/proposalStatus/CreateUpdateProposalStatus.tsx:35]: You could use useDataApiWithFeedback here and then remove error displaying in lines :57-:62

Comment (martin-trajanovski) [reply src/components/settings/proposalStatus/CreateUpdateProposalStatus.tsx:35]: Thanks @jekabs-karklins I saw it and it is coming in the next PR that I am currently working on https://github.com/UserOfficeProject/user-office-frontend/blob/d558d2f36ca6412b2fcfc9da98da3c89d1bf8c8d/src/components/settings/proposalStatus/CreateUpdateProposalStatus.tsx#L33

**Pull request merged**